libshaderc_util/src/compiler.cc

```diff
@@ -199,25 +213,31 @@ bool Compiler::Compile(
   success &= PrintFilteredErrors(error_tag, error_stream, warnings_as_errors_,
                                  suppress_warnings_, program.getInfoLog(),
                                  total_warnings, total_errors);
-  if (!success) return false;
+  if (!success) return result_tuple;
 
-  std::vector<uint32_t> spirv;
+  // 'spirv' is an alias for the compilation_output_data. This alias is added to
+  // serve as an input for the call to DissassemblyBinary.
+  std::vector<uint32_t>& spirv = compilation_output_data;
```

Comment:
This is ok, but you could have just used compilation_output_data in the call to DisassembleBinary instead of making the new variable "spirv".